2.2.0 killing container on SyntaxError

Not sure when exactly this is being thrown in the poller. But it's causing the container to constantly restart as well. Is anyone else running into this problem on 2.2.0? Make sure if you're using helm to install that you update the image to point to 2.2.0. it's currently pointing to 2.1.0.

```
SyntaxError: Unexpected token '%'
    at actualWrite (/app/node_modules/sonic-boom/index.js:242:3)
    at SonicBoom.write (/app/node_modules/sonic-boom/index.js:142:5)
    at Pino.write (/app/node_modules/pino/lib/proto.js:109:43)
    at Pino.LOG [as info] (/app/node_modules/pino/lib/tools.js:36:26)
    at Poller._poll (/app/lib/poller.js:110:18)
    at listOnTimeout (internal/timers.js:531:17)
    at processTimers (internal/timers.js:475:7)
npm info lifecycle kubernetes-external-secrets@2.2.0~start: Failed to exec start script
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! kubernetes-external-secrets@2.2.0 start: `./bin/daemon.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the kubernetes-external-secrets@2.2.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
npm timing npm Completed in 564066ms

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/node/.npm/_logs/2019-11-15T15_11_16_694Z-debug.log
```

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
     "lodash.merge": "^4.6.2",
     "make-promises-safe": "^5.0.0",
     "node-vault": "^0.9.8",
-    "pino": "^5.12.0",
+    "pino": "^5.13.6",
     "prom-client": "^11.5.3"
   },
   "devDependencies": {

